third_party/libtock-drivers/src/usb_ctap_hid.rs

```diff
@@ -71,6 +71,24 @@ pub enum UsbInterface {
     VendorHid = 1,
 }
 
+impl UsbInterface {
+  fn from_endpoint(endpoint: usize) -> Option<Self> {
```

Comment:
Likewise, this could be standard `try_from(usize)` if `SendOrRecvStatus` was an actual `Result`. But that looks like a bigger change and not sure if it's clearer, so this is fine with me.

Reply:
Ignored this as it is a bigger changed.